boomerang.js

```diff
@@ -3671,7 +3671,7 @@ BOOMR_check_doc_domain();
 
 
 
-	delete BOOMR_start;
+	BOOMR_start = null;
```

Comment:
we actually do want to delete it here.  This is a global variable in the main window and we don't want to keep global variables around.  We cannot refer to it as `window.BOOMR_start` unfortunately because that throws an exception in old versions of internet explorer.

Reply:
It is getting deleted and eventually garbage collected. Please see the documentation I linked.

Reply:
But we aren't concerned with garbage collection. We want to remove the variable name from the global space. Setting it to null does not do that. On browsers that do not support strict mode, it does in fact remove the variable name from the global space.

Reply:
I guess I failed to realize that would be a concern. Thanks for pointing that out. I modified my code a bit to distinguish between the strict-friendly deletes and non-strict friendly deletes in older IE browsers. That way it should be painless to refactor the code:

* once it is decided that older versions of IE should no longer be supported
* or when working with forked versions of the code